### DISCUSSION TITLE
How to display a custom field in a template based on its value?

### DISCUSSION THREAD

**Discussion by lintonhale** [liedekef/events-made-easy#634] [General]
Hi Franky,

On this page
https://learnsonomacounty.com/all-classes/
we'd like where it says "$0" to say "Free!" instead.

[eme_if tag='#ESC_CUSTOMFIELD{cost}' = '0']
Free!
[/eme_if]
[eme_if tag='#ESC_CUSTOMFIELD{cost}' <> '0']
$#_FIELD{cost}
[/eme_if]

Is there a way to display a field based on the value of its content?

Thanks!

**Comment by liedekef** [comment]
If 'cost' is your customfield, then:

```
[eme_if tag='#ESC_FIELD{cost}' eq='0']
Free!
[/eme_if]
[eme_if tag='#ESC_FIELD{cost}' ne='0']
#_FIELD{cost}
[/eme_if]
```
